sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -878,21 +879,22 @@ mod checked {
             metrics.clone(),
             move_vm,
             temporary_store,
-            tx_ctx,
+            tx_ctx.clone(),
             gas_charger,
             advance_epoch_pt,
         );
+        tx_ctx.add_ids_created(temporary_store.objects_created_count() as u64);
```

Comment:
we could do this operation according to the result but I figured this is the simplest way to do it and it should be safe, thought?

Reply:
Seems fine to me